test/find.test.js

```diff
@@ -16,6 +16,7 @@ describe('record retrival', function() {
     });
 
     afterAll(function() {
+        delete global.x;
```

Comment:
This should probably delete `global.window`

Reply:
Opps thanks fixed.